clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -6629,8 +6629,9 @@ def warn_counted_by_attr_elt_type_unknown_size :
 
 def err_builtin_get_counted_by_has_side_effects : Error<
   "__builtin_get_counted_by cannot have side-effects">;
-def err_builtin_get_counted_by_must_be_pointer : Error<
-  "__builtin_get_counted_by argument must be a pointer">;
+def err_builtin_get_counted_by_must_be_flex_array_member : Error<
+  "__builtin_get_counted_by argument must be a pointer "
+  "to a flexible array member">;
```

Comment:
```suggestion
  "argument of '__builtin_get_counted_by' must reference "
  "flexible array member">;
```
Something like this maybe? ‘pointer to a FAM’ is technically incorrect since it’s not a pointer *to* a FAM; it’a FAM that decays to a pointer as soon as you try to do pretty much anything with it (including passing it to this builtin).

(I guess you could maybe call it a ‘pointer to’ one in the `&p->x[0]` case, but I think just not including that in the diagnostic should be fine).

Reply:
Also, I only noticed this now, but language constructs (e.g. names of builtin functions) should be enclosed in single quotes in diagnostics (if you pass a `DeclarationName` to a diagnostic, it’ll already be quoted, but that’s not the case here).

Reply:
I looked at other places where a __builtin was referenced and they didn't have quotes around it, but I suppose it couldn't hurt. I'm not crazy about the wording, but I'll think of something.